src/wallet/wallet.h

```diff
@@ -177,28 +178,33 @@ class CMerkleTx : public CTransaction
      */
     int nIndex;
 
-    CMerkleTx()
+    template<typename... Args>
+    CMerkleTx(Args&&... args)
     {
+        SetTx(std::forward<Args>(args)...);
         Init();
     }
 
-    CMerkleTx(const CTransaction& txIn) : CTransaction(txIn)
-    {
-        Init();
-    }
+    operator const CTransaction&() const { return *tx; }
```

Comment:
This is an implicit conversion operator. It allows you to pass a CWalletTx in any place where a 'const CTransaction&' is expected. I can avoid it by changing a number of extra lines.